# Ooyama Yosiyuki <qqke6wd9k@apricot.ocn.ne.jp>, 2015. #zanata
# Ludek Janda <ljanda@redhat.com>, 2018. #zanata
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2018-10-15 09:48+0200\n"
"PO-Revision-Date: 2018-09-11 12:23+0000\n"
"Last-Translator: Ooyama Yosiyuki <qqke6wd9k@apricot.ocn.ne.jp>\n"
"Language-Team: Japanese\n"
"Language: ja\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=1; plural=0\n"
"X-Generator: Zanata 4.6.2\n"

#: ../plugins/reposync.orig.py:42 ../plugins/reposync.py:52
msgid "download all packages from remote repo"
msgstr "リモート repo からすべてのパッケージをダウンロードします"

#: ../plugins/reposync.orig.py:48 ../plugins/reposync.py:62
msgid "download only packages for this ARCH"
msgstr "この ARCH 向けのパッケージのみをダウンロード"

#: ../plugins/reposync.orig.py:50 ../plugins/reposync.py:64
msgid "delete local packages no longer present in repository"
msgstr "リポジトリーにもはや存在しないローカルパッケージを削除"

#: ../plugins/reposync.orig.py:52 ../plugins/reposync.py:66
msgid "also download comps.xml"
msgstr "comps.xml もダウンロード"

#: ../plugins/reposync.orig.py:54 ../plugins/reposync.py:70
msgid "download only newest packages per-repo"
msgstr "最新のパッケージ per-repo のみをダウンロード"

#: ../plugins/reposync.orig.py:56 ../plugins/reposync.py:72
msgid "where to store downloaded repositories "
msgstr "ダウンロード済みリポジトリーの保管場所"

#: ../plugins/reposync.orig.py:58 ../plugins/reposync.py:74
msgid "operate on source packages"
msgstr "ソースパッケージでの操作"

#: ../plugins/reposync.orig.py:98 ../plugins/reposync.py:139
#, python-format
msgid "[DELETED] %s"
msgstr "[DELETED] %s"

#: ../plugins/reposync.orig.py:100 ../plugins/reposync.py:141
#, python-format
msgid "failed to delete file %s"
msgstr "ファイル %s の削除に失敗しました"

#: ../plugins/reposync.orig.py:110
#, python-format
msgid "Could not make repository directory: %s"
msgstr "リポジトリーディレクトリーを作成できませんでした: %s "

#: ../plugins/reposync.orig.py:114 ../plugins/reposync.py:148
#, python-format
msgid "comps.xml for repository %s saved"
msgstr "リポジトリー %s の comps.xml が保存されました"

#: ../plugins/debuginfo-install.py:58
msgid "install debuginfo packages"
msgstr "debuginfo パッケージのインストール"

#: ../plugins/debuginfo-install.py:85
#, python-format
msgid "No match for argument: %s"
msgstr "一致した引数がありません: %s"

#: ../plugins/debuginfo-install.py:92
msgid "No debuginfo packages available to install"
msgstr ""

#: ../plugins/debuginfo-install.py:95
msgid "Unable to find a match"
msgstr "一致するものが見つかりません"

#: ../plugins/debuginfo-install.py:132
#, python-format
msgid "Could not find debuginfo for package: %s"
msgstr ""

#: ../plugins/migrate.py:45
msgid "migrate yum's history, group and yumdb data to dnf"
msgstr "yum の履歴、グループ、および yumdb データを dnf へ移行"

#: ../plugins/migrate.py:54
msgid "Migrating history data..."
msgstr "履歴データを移行中..."

#: ../plugins/repograph.py:50
msgid "Output a full package dependency graph in dot format"
msgstr "ドット形式でパッケージの依存関係グラフ全体を出力"

#: ../plugins/repograph.py:110
#, python-format
msgid "Nothing provides: '%s'"
msgstr "何も提供しません: '%s'"

#: ../plugins/versionlock.py:32
#, python-format
msgid "Unable to read version lock configuration: %s"
msgstr "バージョンロック設定の読み込みができません: %s"

#: ../plugins/versionlock.py:33
msgid "Locklist not set"
msgstr "ロックリストが設定されていません"

#: ../plugins/versionlock.py:34
msgid "Adding versionlock on:"
msgstr "versionlock を追加:"

#: ../plugins/versionlock.py:35
msgid "Adding exclude on:"
msgstr "除外を追加:"

#: ../plugins/versionlock.py:36
msgid "Deleting versionlock for:"
msgstr "versionlock を削除:"

#: ../plugins/versionlock.py:37
msgid "No package found for:"
msgstr "パッケージが見つかりませんでした:"

#: ../plugins/versionlock.py:38
msgid "Excludes from versionlock plugin were not applied"
msgstr "versionlock プラグインからの除外は適用されませんでした"

#: ../plugins/versionlock.py:39
msgid "Versionlock plugin: number of lock rules from file \"{}\" applied: {}"
msgstr "versionlock プラグイン: ファイル \"{}\" のロックルールの数を適用: {}"

#: ../plugins/versionlock.py:40
msgid "Versionlock plugin: number of exclude rules from file \"{}\" applied: {}"
msgstr "versionlock プラグイン: ファイル \"{}\" の除外ルールの数を適用: {}"

#: ../plugins/versionlock.py:41
msgid "Versionlock plugin: could not parse pattern:"
msgstr "versionlock プラグイン: パターンを解析できませんでした:"

#: ../plugins/versionlock.py:119
msgid "control package version locks"
msgstr "パッケージバージョンロックの制御"

#: ../plugins/builddep.py:68
msgid "[PACKAGE|PACKAGE.spec]"
msgstr "[PACKAGE|PACKAGE.spec]"

#: ../plugins/builddep.py:79
#, python-format
msgid "'%s' is not of the format 'MACRO EXPR'"
msgstr "'%s' は、'MACRO EXPR' の形式ではありません"

#: ../plugins/builddep.py:84
msgid "packages with builddeps to install"
msgstr "インストールする builddeps パッケージ"

#: ../plugins/builddep.py:87
msgid "define a macro for spec file parsing"
msgstr "スペックファイルの解析にマクロを定義"

#: ../plugins/builddep.py:90
msgid "treat commandline arguments as spec files"
msgstr "コマンドラインの引数をスペックファイルとして処理"

#: ../plugins/builddep.py:92
msgid "treat commandline arguments as source rpm"
msgstr "コマンドラインの引数をソース rpm として処理"

#: ../plugins/builddep.py:138
msgid "Some packages could not be found."
msgstr "一部のパッケージは見つかりませんでした。"

#. No provides, no files
#: ../plugins/builddep.py:157
#, python-format
msgid "No matching package to install: '%s'"
msgstr "インストール用の一致するパッケージがありません: '%s'"

#: ../plugins/builddep.py:174
#, python-format
msgid "Failed to open: '%s', not a valid source rpm file."
msgstr "開くことに失敗しました: '%s'、有効なソース rpm ファイルではありません。"

#: ../plugins/builddep.py:187 ../plugins/builddep.py:203
#: ../plugins/builddep.py:220
msgid "Not all dependencies satisfied"
msgstr "すべての依存関係が満たされているわけではない"

#: ../plugins/builddep.py:194
#, python-format
msgid "Failed to open: '%s', not a valid spec file: %s"
msgstr "開くことに失敗しました: '%s'、有効なスペックファイルではありません: %s"

#: ../plugins/builddep.py:213 ../plugins/repoclosure.py:118
#, python-format
msgid "no package matched: %s"
msgstr "一致するパッケージはありません: %s"

#: ../plugins/config_manager.py:35
msgid "manage dnf configuration options and repositories"
msgstr "dnf 設定オプションおよびリポジトリーの管理"

#: ../plugins/config_manager.py:41
msgid "repo to modify"
msgstr "修正する repo"

#: ../plugins/config_manager.py:44
msgid "save the current options (useful with --setopt)"
msgstr "現在のオプションを保存 (--setopt で有用)"

#: ../plugins/config_manager.py:47
msgid "enable the specified repos (automatically saves)"
msgstr "指定された repo を有効化 (自動的に保存)"

#: ../plugins/config_manager.py:50
msgid "disable the specified repos (automatically saves)"
msgstr "指定された repo を無効化 (自動的に保存)"

#: ../plugins/config_manager.py:53
msgid "add (and enable) the repo from the specified file or url"
msgstr "指定されたファイルまたは url から repo を追加 (および有効化)"

#: ../plugins/config_manager.py:56
msgid "print current configuration values to stdout"
msgstr "stdout に現在の設定値を印刷"

#: ../plugins/config_manager.py:59
msgid "print variable values to stdout"
msgstr "stdout に変数値を印刷"

#: ../plugins/config_manager.py:76
msgid "Error: Trying to enable and disable repos at the same time."
msgstr "エラー: repo の有効化と無効化を同時に実行します。"

#: ../plugins/config_manager.py:79
msgid "Error: Trying to enable already enabled repos."
msgstr "エラー: すでに有効化された repo を有効化します。"

#: ../plugins/config_manager.py:118
#, python-format
msgid "No matching repo to modify: %s."
msgstr "修正用の一致する repo はありません: %s."

#: ../plugins/config_manager.py:149
#, python-format
msgid "Adding repo from: %s"
msgstr "repo の追加: %s"

#: ../plugins/config_manager.py:173
msgid "Configuration of repo failed"
msgid_plural "Configuration of repos failed"
msgstr[0] "repo の設定に失敗しました"

#: ../plugins/config_manager.py:183
#, python-format
msgid "Could not save repo to repofile %s: %s"
msgstr "repofile %s に repo を保存できませんでした: %s"

#: ../plugins/local.py:117
msgid "Unable to create a directory '{}' due to '{}'"
msgstr "'{}' のため、ディレクトリー '{}' を作成できませんでした"

#: ../plugins/local.py:121
msgid "'{}' is not a directory"
msgstr "'{}' はディレクトリーではありません"

#: ../plugins/local.py:130
msgid "Copying '{}' to local repo"
msgstr "ローカル repo に '{}' をコピー中"

#: ../plugins/local.py:136
msgid "Can't write file '{}'"
msgstr "ファイル '{}' を書き込みできません"

#: ../plugins/local.py:151
msgid "Rebuilding local repo"
msgstr "ローカル repo を再ビルド中"

#: ../plugins/leaves.py:32
msgid "List installed packages not required by any other package"
msgstr "他のパッケージから必要とされないインスール済みパッケージを一覧表示します"

#: ../plugins/needs_restarting.py:167
msgid "determine updated binaries that need restarting"
msgstr "再起動が必要な更新済みバイナリーを決定します"

#: ../plugins/needs_restarting.py:172
msgid "only consider this user's processes"
msgstr "このユーザーのプロセスのみを検討します"

#: ../plugins/repoclosure.py:42
msgid "Display a list of unresolved dependencies for repositories"
msgstr "リポジトリーの未解決の依存関係の一覧を表示します"

#: ../plugins/repoclosure.py:66
msgid "Repoclosure ended with unresolved dependencies."
msgstr "repoclosure は未解決の依存関係で終了しました。"

#: ../plugins/repoclosure.py:153
msgid "check packages of the given archs, can be specified multiple times"
msgstr "特定の arch のパッケージを確認します。複数回指定することができます"

#: ../plugins/repoclosure.py:156
msgid "Specify repositories to check"
msgstr "確認するリポジトリーを指定します"

#: ../plugins/repoclosure.py:158
msgid "Check only the newest packages in the repos"
msgstr "repo の最新パッケージのみを確認します"

#: ../plugins/repoclosure.py:161
msgid "Check closure for this package only"
msgstr "このパッケージのみのクロージャーを確認します"

#: ../plugins/reposync.py:68
msgid "download all the metadata."
msgstr ""

#: ../plugins/reposync.py:121
msgid "Download target '{}' is outside of download path '{}'."
msgstr ""

#: ../plugins/repomanage.py:44
msgid "Manage a directory of rpm packages"
msgstr "rpm パッケージのディレクトリーを管理します"

#: ../plugins/repomanage.py:58
msgid "Pass either --old or --new, not both!"
msgstr "--old または --new のいずれかを渡します。両方ではありません。"

#: ../plugins/repomanage.py:68
msgid "No files to process"
msgstr "処理するファイルはありません"

#: ../plugins/repomanage.py:73
msgid "Could not open {}"
msgstr "{} を開くことができません"

#: ../plugins/repomanage.py:130
msgid "Print the older packages"
msgstr "古いパッケージを印刷します"

#: ../plugins/repomanage.py:132
msgid "Print the newest packages"
msgstr "最新のパッケージを印刷します"

#: ../plugins/repomanage.py:134
msgid "Space separated output, not newline"
msgstr "スペースで区切られた出力で、改行ではありません"

#: ../plugins/repomanage.py:136
msgid "Newest N packages to keep - defaults to 1"
msgstr "維持する最新の N パッケージ - デフォルトは 1 に設定されます"

#: ../plugins/repomanage.py:139
msgid "Path to directory"
msgstr "ディレクトリーへのパス"

#: ../plugins/download.py:41
msgid "Download package to current directory"
msgstr "現在のディレクトリーにパッケージをダウンロードします"

#: ../plugins/download.py:51
msgid "packages to download"
msgstr "ダウンロードするパッケージ"

#: ../plugins/download.py:54
msgid "download the src.rpm instead"
msgstr "代わりに src.rpm をダウンロードします"

#: ../plugins/download.py:56
msgid "download the -debuginfo package instead"
msgstr "代わりに -debuginfo パッケージをダウンロードします"

#: ../plugins/download.py:59
msgid "limit  the  query to packages of given architectures."
msgstr "特定のアーキテクチャーのパッケージへのクエリーを制限します。"

#: ../plugins/download.py:61
msgid "resolve and download needed dependencies"
msgstr "必要な依存関係を解決し、ダウンロードします"

#: ../plugins/download.py:63
msgid ""
"when running with --resolve, download all dependencies (do not exclude "
"already installed ones)"
msgstr ""

#: ../plugins/download.py:66
msgid ""
"print list of urls where the rpms can be downloaded instead of downloading"
msgstr "ダウンロードする代わりに、rpm をダウンロードできる url の一覧を印刷します"

#: ../plugins/download.py:71
msgid "when running with --url, limit to specific protocols"
msgstr "--url で実行中の際は、特定のプロトコルに限定します"

#: ../plugins/download.py:110
#, python-format
msgid "Failed to get mirror for package: %s"
msgstr "パッケージのミラー取得に失敗しました: %s"

#: ../plugins/download.py:193
msgid "Exiting due to strict setting."
msgstr "厳密な設定により終了中です。"

#: ../plugins/download.py:213
msgid "Error in resolve of packages:"
msgstr "パッケージの解決でエラー:"

#: ../plugins/download.py:230
#, python-format
msgid "No source rpm defined for %s"
msgstr "%s に対して定義されているソース rpm はありません"

#: ../plugins/download.py:248 ../plugins/download.py:262
#, python-format
msgid "No package %s available."
msgstr "利用可能なパッケージ %s はありません。"

#: ../plugins/show_leaves.py:54
msgid "New leaves:"
msgstr "新規のリーフパッケージ (他のパッケージから依存されていないパッケージ) :"

#: ../plugins/copr.py:54
msgid "yes"
msgstr "はい"

#: ../plugins/copr.py:54
msgid "y"
msgstr "y"

#: ../plugins/copr.py:55
msgid "no"
msgstr "いいえ"

#: ../plugins/copr.py:55
msgid "n"
msgstr "n"

#: ../plugins/copr.py:74
msgid "Interact with Copr repositories."
msgstr "Copr リポジトリーとの対話。"

#: ../plugins/copr.py:75
msgid ""
"\n"
"  enable name/project [chroot]\n"
"  disable name/project\n"
"  remove name/project\n"
"  list --installed/enabled/disabled\n"
"  list --available-by-user=NAME\n"
"  search project\n"
"\n"
"  Examples:\n"
"  copr enable rhscl/perl516 epel-6-x86_64\n"
"  copr enable ignatenkobrain/ocltoys\n"
"  copr disable rhscl/perl516\n"
"  copr remove rhscl/perl516\n"
"  copr list --enabled\n"
"  copr list --available-by-user=ignatenkobrain\n"
"  copr search tests\n"
"    "
msgstr ""
"\n"
"  enable name/project [chroot]\n"
"  disable name/project\n"
"  remove name/project\n"
"  list --installed/enabled/disabled\n"
"  list --available-by-user=NAME\n"
"  search project\n"
"\n"
"  Examples:\n"
"  copr enable rhscl/perl516 epel-6-x86_64\n"
"  copr enable ignatenkobrain/ocltoys\n"
"  copr disable rhscl/perl516\n"
"  copr remove rhscl/perl516\n"
"  copr list --enabled\n"
"  copr list --available-by-user=ignatenkobrain\n"
"  copr search tests\n"
"    "

#: ../plugins/copr.py:101
msgid "List all installed Copr repositories (default)"
msgstr "インストール済みのすべての Copr リポジトリーを一覧表示します (デフォルト)"

#: ../plugins/copr.py:103
msgid "List enabled Copr repositories"
msgstr "有効化された Copr リポジトリーを一覧表示します"

#: ../plugins/copr.py:105
msgid "List disabled Copr repositories"
msgstr "無効化された Copr リポジトリーを一覧表示します"

#: ../plugins/copr.py:107
msgid "List available Copr repositories by user NAME"
msgstr "利用可能な Copr リポジトリーをユーザー NAME ごとに一覧表示します"

#: ../plugins/copr.py:109
msgid "Specify an instance of Copr to work with"
msgstr ""

#: ../plugins/copr.py:143 ../plugins/copr.py:208 ../plugins/copr.py:228
msgid "Error: "
msgstr "エラー: "

#: ../plugins/copr.py:144
msgid ""
"specify Copr hub either with `--hub` or using "
"`copr_hub/copr_username/copr_projectname` format"
msgstr ""

#: ../plugins/copr.py:147
msgid "multiple hubs specified"
msgstr ""

#: ../plugins/copr.py:209 ../plugins/copr.py:213
msgid "exactly two additional parameters to copr command are required"
msgstr "copr コマンドに厳密に 2 つの追加パラメーターが必要です"

#: ../plugins/copr.py:229
msgid "use format `copr_username/copr_projectname` to reference copr project"
msgstr "copr プロジェクトを参照するには `copr_username/copr_projectname` 形式を使用します"

#: ../plugins/copr.py:231
msgid "bad copr project format"
msgstr "不正な copr プロジェクト形式"

#: ../plugins/copr.py:245
#, python-brace-format
msgid ""
"\n"
"You are about to enable a Copr repository. Please note that this\n"
"repository is not part of the main distribution, and quality may vary.\n"
"\n"
"The Fedora Project does not exercise any power over the contents of\n"
"this repository beyond the rules outlined in the Copr FAQ at\n"
"<https://docs.pagure.org/copr.copr/user_documentation.html#what-i-can-build-in-copr>,\n"
"and packages are not held to any quality or security level.\n"
"\n"
"Please do not file bug reports about these packages in Fedora\n"
"Bugzilla. In case of problems, contact the owner of this repository.\n"
"\n"
"Do you really want to enable {0}?"
msgstr ""

#: ../plugins/copr.py:261
msgid "Repository successfully enabled."
msgstr "リポジトリが正常に有効化されました。"

#: ../plugins/copr.py:265
msgid "Repository successfully disabled."
msgstr "リポジトリが正常に無効化されました。"

#: ../plugins/copr.py:269
msgid "Repository successfully removed."
msgstr "リポジトリーが正常に削除されました。"

#: ../plugins/copr.py:273 ../plugins/copr.py:606
msgid "Unknown subcommand {}."
msgstr "不明なサブコマンド {}。"

#: ../plugins/copr.py:322
msgid ""
"* These coprs have repo file with an old format that contains no information"
" about Copr hub - the default one was assumed. Re-enable the project to fix "
"this."
msgstr ""

#: ../plugins/copr.py:334
msgid "Can't parse repositories for username '{}'."
msgstr "ユーザー名 '{}' のリポジトリーを解析できません。"

#: ../plugins/copr.py:337
msgid "List of {} coprs"
msgstr "{} coprs の一覧"

#: ../plugins/copr.py:345
msgid "No description given"
msgstr "説明がありません"

#: ../plugins/copr.py:357
msgid "Can't parse search for '{}'."
msgstr "'{}' の検索を解析できません。"

#: ../plugins/copr.py:360
msgid "Matched: {}"
msgstr "一致: {}"

#: ../plugins/copr.py:368
msgid "No description given."
msgstr "説明が与えられていません。"

#: ../plugins/copr.py:381
msgid "Safe and good answer. Exiting."
msgstr "安全で優れた回答。終了中。"

#: ../plugins/copr.py:388
msgid "This command has to be run under the root user."
msgstr "このコマンドは root ユーザーの下で実行する必要があります。"

#: ../plugins/copr.py:449
msgid ""
"This repository does not have any builds yet so you cannot enable it now."
msgstr "このリポジトリーにはまだビルドがありませんので、今すぐ有効化できません。"

#: ../plugins/copr.py:452
msgid "Such repository does not exist."
msgstr "そのようなリポジトリーは存在しません。"

#: ../plugins/copr.py:496
#, python-brace-format
msgid "Failed to remove copr repo {0}/{1}/{2}"
msgstr ""

#: ../plugins/copr.py:507
msgid "Failed to disable copr repo {}/{}"
msgstr "copr repo {}/{} の無効化に失敗しました。"

#: ../plugins/copr.py:523 ../plugins/copr.py:561
msgid "Unknown response from server."
msgstr "サーバーからの不明な応答です。"

#: ../plugins/copr.py:545
msgid "Interact with Playground repository."
msgstr "Playground リポジトリーとの対話。"

#: ../plugins/copr.py:550
msgid ""
"\n"
"You are about to enable a Playground repository.\n"
"\n"
"Do you want to continue?"
msgstr ""
"\n"
"Playground リポジトリーを有効化しようとしています。\n"
"\n"
"続行しますか?"

#: ../plugins/copr.py:596
msgid "Playground repositories successfully enabled."
msgstr "Playground リポジトリーが正常に有効化されました。"

#: ../plugins/copr.py:599
msgid "Playground repositories successfully disabled."
msgstr "Playground リポジトリーが正常に無効化されました。"

#: ../plugins/copr.py:603
msgid "Playground repositories successfully updated."
msgstr "Playground リポジトリーが正常に更新されました。"

#: ../plugins/debug.py:53
msgid "dump information about installed rpm packages to file"
msgstr "ファイルにインストール済みの rpm パッケージに関するダンプ情報"

#: ../plugins/debug.py:67
msgid "do not attempt to dump the repository contents."
msgstr "リポジトリーコンテンツのダンプは試みないでください。"

#: ../plugins/debug.py:70
msgid "optional name of dump file"
msgstr "ダンプファイルの名前のオプション"

#: ../plugins/debug.py:95
#, python-format
msgid "Output written to: %s"
msgstr "書き込まれた出力: %s"

#: ../plugins/debug.py:172
msgid "restore packages recorded in debug-dump file"
msgstr "debug-dump ファイルに記録されたパッケージを復元します"

#: ../plugins/debug.py:183
msgid "output commands that would be run to stdout."
msgstr "stdout に実行するコマンドを出力します。"

#: ../plugins/debug.py:186
msgid "Install the latest version of recorded packages."
msgstr "記録されたパッケージの最新バージョンをインストールします。"

#: ../plugins/debug.py:189
msgid ""
"Ignore architecture and install missing packages matching the name, epoch, "
"version and release."
msgstr "アーキテクチャーを無視し、名前、エポック、バージョン、およびリリースと一致する不足のパッケージをインストールします。"

#: ../plugins/debug.py:194
msgid "limit to specified type"
msgstr "指定したタイプに限定します"

#: ../plugins/debug.py:196
msgid "name of dump file"
msgstr "ダンプファイルの名前"

#: ../plugins/debug.py:264
#, python-format
msgid "Package %s is not available"
msgstr "パッケージ %s は利用できません"

#: ../plugins/debug.py:274
#, python-format
msgid "Bad dnf debug file: %s"
msgstr "不正な dnf デバッグファイル: %s"
